### ISSUE TITLE
Plugin not working after 20.0.1 update

### ISSUE THREAD

**Issue by ABEIDO** [hyperion-project/hyperion.kodi#56]
Hi

Latest version is not working for me.

All settings ok but after update i just get a Ambilight error at Kodi start and info to check log

Tested to rollback to 1.0.6 and then everything works fine.

Using
Windows 10
Kodi 20.2
Hyperion.NG 2.0.15

Log:
```
2023-11-20 18:15:03.856 T:4372    debug <general>: Thread waiting 4372 terminating
2023-11-20 18:15:03.859 T:5148    debug <general>: Thread BackgroundLoader start, auto delete: false
2023-11-20 18:15:03.860 T:6200    debug <general>: ------ Window Init (DialogNotification.xml) ------
2023-11-20 18:15:03.865 T:5148    debug <general>: Thread BackgroundLoader 5148 terminating
2023-11-20 18:15:04.087 T:9024    error <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'TypeError'>
                                                   Error Contents: 'type' object is not subscriptable
                                                   Traceback (most recent call last):
                                                     File "C:\Users\user\AppData\Roaming\Kodi\addons\script.service.hyperion\service.py", line 29, in <module>
                                                       from resources.lib.monitor import HyperionMonitor
                                                     File "C:\Users\user\AppData\Roaming\Kodi\addons\script.service.hyperion\resources\lib\monitor.py", line 37, in <module>
                                                       class HyperionMonitor(xbmc.Monitor):
                                                     File "C:\Users\user\AppData\Roaming\Kodi\addons\script.service.hyperion\resources\lib\monitor.py", line 117, in HyperionMonitor
                                                       def get_capture_size(self) -> tuple[tuple[int, int], int]:
                                                   TypeError: 'type' object is not subscriptable
                                                   -->End of Python script error report<--
                                                   
2023-11-20 18:15:04.087 T:9024    debug <general>: CPythonInvoker::onExecutionDone(80, C:\Users\user\AppData\Roaming\Kodi\addons\script.service.hyperion\service.py)
2023-11-20 18:15:04.087 T:9024    debug <general>: .
2023-11-20 18:15:04.281 T:9024    debug <general>: Python interpreter stopped
2023-11-20 18:15:04.281 T:9024    debug <general>: Thread LanguageInvoker 9024 terminating
```

**Comment by CiniBN** [comment]
I get the same error.

**Comment by ABEIDO** [comment]
To add it happens also on fresh Kodi install on Linux. So i cant see any 3 party issue casuing this.

**Comment by sanderkooger** [comment]
HEre 2 just installed. Hyperion is running on a sept server and is working. but the addon does not want to send data.

**Comment by bachulator** [comment]
Any chances to see solution for the problem?

**Comment by Lord-Grey** [comment]
Hi @sanzoghenzo 
Seems I let this one slip...
Would you be so kind and have a look at the issue and advise what needs to be corrected, please?

**Comment by sanzoghenzo** [comment]
Hi @Lord-Grey,

This is a python version "problem", I was convinced that kodi 20 would use python 3.10 on all the platforms, but it doesn't.

From a quick look at the source the fix should be as easy as adding `from typing import Tuple` and then use the `Tuple` (with capital T) in [this line](https://github.com/hyperion-project/hyperion.kodi/blob/6c2517703d5b956b9571264f9ebeea1c8af53285/script.service.hyperion/resources/lib/monitor.py#L117) instead of the lowercase one.

I don't see any other places where the typing annotations could cause errors in older python versions.

Unfortunately I don't have the setup to make the changes and test them since I've dropped my living room "ambilight" project (for now, who knows what tomorrow brings!!)

**Comment by Lord-Grey** [comment]
Published a new release v20.0.2 for testing [here](https://github.com/hyperion-project/hyperion.kodi/releases/tag/v20.0.2) 

@sanzoghenzo Thanks for your continuous support!!!

**Comment by ABEIDO** [comment]
Hi, i tested 20.0.2 on my Windows Kodi. 

Worked perfect, thx for fix.
